packages/common/container-definitions/src/deltas.ts

```diff
@@ -156,20 +156,6 @@ export interface IDeltaManagerEvents extends IEvent {
 export interface IDeltaManager<T, U>
 	extends IEventProvider<IDeltaManagerEvents>,
 		IDeltaSender {
-	/**
-	 * The queue of inbound delta messages
-	 * @deprecated Do not use, for internal use only. There are a lot of complications in core pieces of the runtime
-	 * may break if this is used directly. For example summarization and op processing.
-	 */
-	readonly inbound: IDeltaQueue<T>;
```

Comment:
Overall - that's a great change.
But there is usage in office-bohemia. It would be great if we could first go into their repo and figure out what needs to be done to get rid of it.

@tyler-cai-microsoft - FYI, as you were looking at it at some point

Reply:
Last time I checked, it was just being used for testing purposes, which can be replaced probably with disconnect and reconnect.

Reply:
I check their repo. There are mostly using it in examples to mimic offline behaviour, that is to go offline and comeback. I think probably, container.disconnect could achieve that?

Reply:
Yeah, container.disconnect is a better solution to achieving that

Reply:
Its usage is now removed from their repo.